3/test/excercise3.spec.js

```diff
@@ -0,0 +1,110 @@
+const { gql } = require('apollo-server')
+const server = require('../server.js')
+const {createTestClient} = require('apollo-server-testing');
+const jwt = require('jsonwebtoken');
+
+const { query, mutate } = createTestClient(server);
+
+describe('Query: Given a list of todos and assignees', () => {
+
+    const queryTodoLength = gql`query getCountTodos {countTodos}`;
+    const queryAssigneeLength = gql`query getCountAssignees {countAssignees}`;
+    
+    it('get all todo items at startup', () => {
+        query({query: queryTodoLength}).then((result) => {
```

Comment:
I would prefer `async/await` syntax. I always feel uncomfortable about ever reaching the `then` block.

Plus, if you use `async/await` in addition to jest async helpers, you get nice readable code:
```js
await expect(query({query: queryTodoLength})).resolves.toMatchObject({
  data: {
    countAssignees: 3
  },
  errors: undefined,
})
```